### ISSUE TITLE
Search on payments by date causes Server error

### ISSUE THREAD

**Issue by RichardWigley** [BCS-io/letting#310]
Searching for a property id, a human reference number not a database table index, results in a server error 500.  

Error Reproduce
* Navigate to payments_by_dates
* In the search enter a house number
* Get error
````
query_by_referrerapp/models/utils/literal_search.rb:48
goapp/models/utils/literal_search.rb:27
indexapp/controllers/search_controller.rb:18

Missing: controller: payments_by_dates, action: index
`````

Expect
* Expect that we get the payments by date for that house number.

**Comment by RichardWigley** [comment]
Fix as part of:  7cb8988c64e53e90b44faa69812404044bc9da36

**Comment by MargaretLangley** [comment]
fix2 ok
